tests/core/conftest.py

```diff
@@ -58,19 +64,35 @@ def admin_mc():
     }, verify=False)
     client = rancher.Client(url=BASE_URL, token=r.json()['token'],
                             verify=False)
-    return ManagementContext(client)
+    k8s_client = kubernetes_api_client(client, 'local')
+    return ManagementContext(client, k8s_client)
+
+
+def kubernetes_api_client(rancher_client, cluster_name):
```

Comment:
Can we keep the fixtures at the top and functions after that? Class > fixture > func

Reply:
Are there linters that will verify this for us?